src/hand_gesture_recognition/notebook/Model_training.ipynb

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
Move the jupyter notebook out of `src` directory. Create separate directory called `train` and put it there